README.md

```diff
@@ -34,3 +36,18 @@ Any of the following will merge scripts:
 * `./mergeScripts.sh`
 * `git commit` runs the above script as a pre-commit hook
 
+The output is in `./bin/roll20.js`
+
+### CSS
+
+[Sass](https://sass-lang.com/guide) is a tool that provides the ability to compile CSS from individual
```

Comment:
Sass is a useful tool for modularizing CSS, and will also help with other topics like nesting and reuse.